common/validator_dir/src/validator_dir.rs

```diff
@@ -80,12 +76,7 @@ impl ValidatorDir {
             return Err(Error::DirectoryDoesNotExist(dir));
         }
 
-        // Lock the keystore file that *might* be in this directory.
```

Comment:
My understanding is that we need the lockfile here so that it cleans up the .lock files on drop?
Removing this might mean the lock files don't get deleted on exit.

Reply:
(might be wrong here though)

Reply:
I'll test this out locally

Reply:
So it worked for me locally.. but I wonder if it has an impact on abrupt shutdowns?

Reply:
Oooo y'know this is probably it

```
impl Drop for Lockfile {
    fn drop(&mut self) {
        let _ = fs::remove_file(&self.path);
    }
}
```

Reply:
Ok added it back in